src/AppKit/Enums.cs

```diff
@@ -2154,10 +2154,191 @@ public enum NSPopUpArrowPosition : ulong {
 		Bottom
 	}
 
-	public static class NSFileTypeForHFSTypeCode {
-		public static readonly string ComputerIcon = "root";
-		public static readonly string DesktopIcon = "desk";
-		public static readonly string FinderIcon = "FNDR";
+	// FileType 4cc values to use with NSFileTypeForHFSTypeCode.
+	public enum HFSTypeCode: uint
```

Comment:
`HfsTypeCode` 
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers